Apromore-Custom-Plugins/Process-Discoverer-Portal-Plugin/src/main/java/org/apromore/plugin/portal/processdiscoverer/components/ViewSettingsController.java

```diff
@@ -144,6 +146,7 @@ public class ViewSettingsController extends VisualController {
     private static final String ON_FORCE_SELECT = "onForceSelect";
 
     private boolean disabled = false;
+    private boolean costDisabled = false;
```

Comment:
Maybe better naming: costCentreNotAvailable?

Reply:
Changed the variable name to isCostCentreAvailable and updated the checks + assignments.